src/NLog/Layouts/CompoundLayout.cs

```diff
@@ -64,9 +64,9 @@ public CompoundLayout()
         /// </summary>
         protected override void InitializeLayout()
         {
-            base.InitializeLayout();
             foreach (var layout in Layouts)
                 layout.Initialize(this.LoggingConfiguration);
+            base.InitializeLayout();
```

Comment:
Since the base.InitializeLayout() performs object-scanning, then I wanted it to be last, so it would find all relevant objects. But I will revert the change.